tests/platform/test_platform_info.py

```diff
@@ -12,12 +12,31 @@
 
 import pytest
 
+from loganalyzer import LogAnalyzer, LogAnalyzerError
+from common.utilities import wait_until
 from psu_controller import psu_controller
-
+from thermal_control_test_helper import *
 
 CMD_PLATFORM_SUMMARY = "show platform summary"
 CMD_PLATFORM_PSUSTATUS = "show platform psustatus"
 CMD_PLATFORM_SYSEEPROM = "show platform syseeprom"
+CMD_PLATFORM_FANSTATUS = "show platform fan"
+CMD_PLATFORM_TEMPER = "show platform temperature"
+
+THERMAL_CONTROL_TEST_WAIT_TIME = 65
+THERMAL_CONTROL_TEST_CHECK_INTERVAL = 5
+
+THERMAL_POLICY_VALID_FILE = 'valid_policy.json'
+THERMAL_POLICY_INVALID_FORMAT_FILE = 'invalid_format_policy.json'
+THERMAL_POLICY_INVALID_VALUE_FILE = 'invalid_value_policy.json'
+
+LOG_EXPECT_POLICY_FILE_INVALID = '.*Caught exception while initializing thermal manager.*'
```

Comment:
I think sometimes we need to reuse log messages in different tests, maybe put those log messages in some common directory?

Reply:
Where to put them?

Reply:
It's up to you.
The reason I have this comment is that I also had the same comment in Noa's PR where ACL related log messages where duplicated from ACL test.
You can sync with her to find out what place can be usable for both PRs.
I think she put them in log_messages file in tests/ folder, but PR is under review, so not merged yet.

Reply:
OK, I will keep watching her PR and move the log messages to the file she created.